app/lib/utils/queryMapper.js

```diff
@@ -21,6 +21,43 @@ function getLocationHeading(query) {
   return undefined;
 }
 
+function getResultsHeading(query, loc) {
+  const location = loc.toUpperCase();
+  if (query.type) {
+    if (comparisons.areEqual(query.type, constants.serviceTypes.professional)
+      && (comparisons.getValues(constants.serviceChoices).includes(query.origin))) {
+      return `Sexual health professionals near '${location}'`;
+    }
+    if (comparisons.areEqual(query.type, constants.serviceTypes.kit)) {
+      if (query.origin === constants.serviceChoices['16to25']) {
+        return `Where you can pick up a free test kit near '${location}'`;
+      } else if (query.origin === constants.serviceChoices.over25) {
+        return `Where you can buy a test near '${location}'`;
+      }
+    }
+  }
+  return undefined;
+}
+
+function getResultsPara(query) {
```

Comment:
Can this function be renamed, along with the associated property `resultsPara` to `resultsParagraph` or better yet, a more descriptive name as to what the paragraph contains or how it is used.